components/ui-tooling-preview/library/build.gradle.kts

```diff
@@ -0,0 +1,55 @@
+import org.jetbrains.kotlin.gradle.ExperimentalKotlinGradlePluginApi
+
+plugins {
+    kotlin("multiplatform")
+    id("maven-publish")
+    id("com.android.library")
+}
+
+val composeVersion = extra["compose.version"] as String
+
+kotlin {
+    @OptIn(ExperimentalKotlinGradlePluginApi::class)
+    targetHierarchy.default()
+    jvm("desktop")
+    androidTarget {
+        publishLibraryVariants("release")
+        compilations.all {
+            kotlinOptions {
+                jvmTarget = "11"
+            }
+        }
+    }
+    iosX64()
+    iosArm64()
+    iosSimulatorArm64()
+    js {
+        browser {
+            testTask(Action {
+                enabled = false
+            })
+        }
+    }
+    macosX64()
+    macosArm64()
+}
+
+android {
+    compileSdk = 34
+    namespace = "org.jetbrains.compose.ui.tooling.preview"
+    defaultConfig {
+        minSdk = 21
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_11
+        targetCompatibility = JavaVersion.VERSION_11
+    }
+}
+
+configureMavenPublication(
+    groupId = "org.jetbrains.compose.components",
```

Comment:
We, users, and IDE devs should be aware that this is subject to change in the future. Please, mention that in the PR message (with mentioning that it is an experiemental library).

Reply:
Done, mentioned